astropy/io/ascii/core.py

```diff
@@ -1427,6 +1427,13 @@ def _get_writer(Writer, fast_writer, **kwargs):
 
     from .fastbasic import FastBasic
 
+    # A values of None for fill_values imply getting the default string
+    # representation of masked values (depending on the writer class), but the
+    # machinery expects a list.  The easiest here is to just pop the value off,
+    # i.e. fill_values=None is the same as not providing it at all.
+    if 'fill_values' in kwargs and kwargs['fill_values'] is None:
```

Comment:
This seems overly verbose, isn't this equivalent to `kwargs.get('fill_values', None) is None`